docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
+However, Polkadot will be a platform for other chains that _do_ implement
+smart contracts. It's possible for parachains to enable smart contract functionality
+and then benefit from the security and interoperability features of Polkadot. 
+Additionally, already existing smart contract chains can connect to Polkadot
+as a parachain, or via a bridge.
 
-The Web3 Foundation will distribute up to 20% of mainnet DOTs prior to network launch (see [Light Paper](https://polkadot.network/Polkadot-lightpaper.pdf) or the [Polkadot Network FAQ](https://polkadot.network/faq/)).  Subscribe to the Polkadot newsletter on [polkadot.network](https://polkadot.network/) for further updates.
+While the Polkadot Relay Chain does not implement smart contracts directly,
+undoubtedly there will be parachains that do. So it's better to say that the 
+Polkadot _ecosystem_ has smart contracts versus "Polkadot has smart contracts."
 
-Mainnet DOT tokens are not transferrable until mainnet launch, expected in early 2020. Any transfers before that time of mainnet DOTs are illegitimate and unauthorized. DOTs can not be moved from a current allocation address. Individuals with an allocation of DOTs who transfer their DOT address to someone else can always keep a copy of their private key, therefore there is extreme risk for individuals participating in transfers of DOTs before mainnet launch.
+### How will the Polkadot Relay Chain connect to external chains in the ecosystem?
 
-### What are the ways to find out the minimum stake necessary for the validators?
+One of the cornerstone interoperability technologies being research and developed
+for deployment on Polkadot is cross-chain bridges. Bridges come in a variety
+of flavors with varying levels of trust associated with them. Polkadot is predominantly
+researching the trust-minimized flavor that imposes economic costs on the operators
+of the bridge, and therefore makes it economically secure. Bridge efforts are being
+work on in concert with other projects in the ecosystem. Eventually, there
+will be bridges between Polkadot and most of the other major chains. 
 
-There are a few ways to estimate it.
+## DOTs
 
-- [Offline Phragmen](https://github.com/kianenigma/offline-phragmen)
+### What is the inflation rate of the DOT?
 
-If you want to know what would be the outcome of the election in the next era, this can provide an estimation result. You can customize parameters such as number of validators to elect, network, and WebSocket endpoint, but it would take some time to build the binary.
-- [Validator stats script](https://github.com/ansonla3/kusama-validator-stats)
+The inflation rate is 10% per year.
```

Comment:
```suggestion
The inflation rate is approximately 10% per year, although this can vary based on how much is staked.
```

Reply:
Mmm it shouldn't. Inflation will be a fixed 10% but the ratio of reward to validators vs treasury will vary depending on how much is staked.